kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/DocumentInstanceList.java

```diff
@@ -29,7 +29,7 @@
 @XmlRootElement(name = "document-instance-list")
 public class DocumentInstanceList implements ItemList<DocumentInstance> {
 
-    @XmlElement(name="document-instance")
+    @XmlElement(name="document-instances")
```

Comment:
Hmm, I thought that the @XmlElement defines the name for individual XML tags (not the wrapping collection tag). What wrapper do you mean? (I ma likely just not seeing something, but would like to understand what is going on :))